ui/imports/shared/status/StatusInputListPopup.qml

```diff
@@ -130,46 +140,52 @@ Popup {
 
     ListView {
         id: listView
-        model: popup.modelList || []
-        keyNavigationEnabled: true
-        Layout.fillHeight: true
-        width: parent.width
+
         anchors.top: searchBox.bottom
         anchors.topMargin: searchBox.visible ? Style.current.smallPadding : 0
         anchors.bottom: parent.bottom
+
+        model: root.modelList || []
+        keyNavigationEnabled: true
+        Layout.fillHeight: true
+        width: parent.width
         clip: true
 
         delegate: Rectangle {
             property variant myData: typeof modelData === "undefined" ? model : modelData
-            property string myText: popup.getText(myData)
+            property string myText: root.getText(myData)
             id: rectangle
             visible: searchBox.text === "" || myText.includes(searchBox.text)
-            color: listView.currentIndex === index ? Style.current.backgroundHover : Style.current.transparent
+            color: listView.currentIndex
+                   === index ? Style.current.backgroundHover : Style.current.transparent
```

Comment:
Just aesthetic, but it would be prettier if the line break was after `?`, that way the condition stays whole

Reply:
This change was reverted.